configs/config-ccm-psk-dtls1_2.h

```diff
@@ -23,19 +23,20 @@
  * See README.txt for usage instructions.
  */
 
+#define MBEDTLS_PSA_CRYPTO_CONFIG_FILE "../configs/crypto-config-ccm-psk-tls1_2.h"
+
+#define MBEDTLS_PSA_CRYPTO_CONFIG
+
 /* System support */
 //#define MBEDTLS_HAVE_TIME /* Optionally used in Hello messages */
 /* Other MBEDTLS_HAVE_XXX flags irrelevant for this configuration */
 
 /* Mbed TLS modules */
-#define MBEDTLS_AES_C
-#define MBEDTLS_CCM_C
 #define MBEDTLS_CIPHER_C
```

Comment:
In all cases we have `MBEDTLS_SSL_TLS_C` set, so CIPHER and MD need to be enabled when `MBEDTLS_USE_PSA_CRYPTO` is disabled (relevant build failures as of [`489688c`](https://github.com/Mbed-TLS/mbedtls/commit/489688c0f7eddf57334d1d7e8ad0f03ba6bf82d0) are caused [here](https://github.com/Mbed-TLS/mbedtls/blob/489688c0f7eddf57334d1d7e8ad0f03ba6bf82d0/include/mbedtls/check_config.h#L865-L882)).

Reply:
Right, but we're dropping the non-PSA tests, so `MBEDTLS_USE_PSA_CRYPTO` should now be always on, right?

Reply:
I spoke to @ronald-cron-arm about dropping the non-PSA tests earlier today. My (possibly mistaken) understanding is that the plan is to keep the non-PSA build in `test-ref-configs` at least for now, as it might not be clear yet how the testing will look post split. As I am regrettably short on time for this work, I think it might be best for me to just push commits which fix all of the uncontroversial mistakes here, and leave the test coverage comments until the plan is more set in stone.